src/appTypes/navigation/lists.ts

```diff
@@ -6,6 +6,7 @@ import { CommonStackParamsList } from './common';
 
 export type PortfolioParamsPortfolio = {
   PortfolioScreen: undefined;
+  SingleGroup: undefined;
```

Comment:
Each Tab has CommonStack inside and CommonStack has Collection screen, that is a renamed version of SingleGroup. You can directly use navigation.navigate('Collection')